### ISSUE TITLE
I haven't excluded any engrams from the mod at all, but i am unable to find some of the engrams. 

### ISSUE THREAD

**Issue by Skadis-Venom** [VoidArkMods/QoLPlus#231]
I haven't excluded any engrams from the mod at all, but I am unable to find some of the engrams. These are the lines i have for this mod. 

[QoLPlus]
AllowMultiToolNeuterAll=False
DisableMultiToolDinoKillMode=True
FarmerRangeInFoundations=50
FarmerSlotCount=500
FridgeSlotCount=300
GardenerRangeInFoundations=50
GardenerSlotCount=500
MaxPlayersPerTribe=6
MutatorModeBlacklist=SwapGender, AssignGender, AllowBreeding, Imprint, FreezeGrowth, BoostGrowth
NannyMaxImprint=100
NannyRangeInFoundations=50
NannySlotCount=300
RemoveFloorRequirementFromStructurePlacement=False
ReplicatorScaleMultiplier=.5
ResourcePullRangeInFoundations=25
TribePropagatorLimit=1

**Comment by bradster223** [comment]
Did you figure this out?

**Comment by Skadis-Venom** [comment]
No, I ended up switching to a different mod though. Thank you for the response. 
    On Wednesday, December 25, 2024 at 12:00:36 PM CST, bradster223 ***@***.***> wrote:  
 
 


Did you figure this out?
